Assignment4_bash.sh

```diff
@@ -0,0 +1,13 @@
+pip-compile --output-file=requirements.txt requirements.in --upgrade
+
+pip3 install -r requirements.txt
+
+pre-commit install
+
+pre-commit run --all-files
+
+read -p "please enter csv name " file
+read -p "please enter the response variable name" response
```

Comment:
There are no fail checks..If user does not enter any input then the code fails

Reply:
thanks for pointing that out. Just added it.